sdks/java/core/src/main/java/org/apache/beam/sdk/util/HistogramData.java

```diff
@@ -74,6 +77,41 @@ public HistogramData(BucketType bucketType) {
     this.sumOfSquaredDeviations = 0;
   }
 
+  /**
+   * Create a histogram from DataflowHistogramValue proto.
+   *
+   * @param histogramProto DataflowHistogramValue proto used to populate stats for the histogram.
+   */
+  public HistogramData(DataflowHistogramValue histogramProto) {
```

Comment:
This seems like Dataflow utility code that could live in runners/dataflow/worker/ somewhere